.rubocop.yml

```diff
@@ -213,6 +213,8 @@ Minitest/RefutePathExists:
   Enabled: true
 Minitest/UnspecifiedException:
   Enabled: true
+Minitest/AssertEmptyLiteral:
+  Enabled: false
```

Comment:
This rule was failing the build. I'm not sure if this is the correct way to deal with that. I am not familiar with rubocop

Reply:
this is the correct way. I'd rather see us disable rules and then address the issues and open a PR with the fixes and re-enablement later.